scripts/setup_sgtm_tasks_project.py

```diff
@@ -25,6 +25,7 @@ class CustomField:
         name="PR Status",
         enum_options=[
             EnumOption(name="Open", color="green"),
+            EnumOption(name="Draft", color="grey"),
```

Comment:
where do we have the list of colors that are options? Just want to double check `"grey"` is an official custom field color

Reply:
🆒